service/lib/agama/storage/config.rb

```diff
@@ -190,6 +190,39 @@ def potential_for_md_device
         drives + drives.flat_map(&:partitions)
       end
 
+      # Encryption configs, excluding encryptions from skipped devices.
+      #
+      # @return [Array<Configs::Encryption>]
+      def valid_encryptions
```

Comment:
I'm not sure about these `valid_whatever`. I had no problem with the name when they were internal methods of a conversion. But now that they are part of the `Storage::Config` API I would be more picky. And I don't think "valid" represents its meaning.

Reply:
I don't like `#valid_whatever` neither, but my brain is not able to nothing better right now :(

Reply:
Some suggestions from the top of my head: `relevant_`, `non_skipped_`, `final_`, `processable_`.